ludwig/automl/base_config.py

```diff
@@ -15,13 +15,15 @@
 """
 
 import os
+from dataclasses import dataclass
 from typing import List, Union
 
 import pandas as pd
+from ludwig.automl.data_source import DataSource, DataframeSource
 from ludwig.automl.utils import (FieldInfo, avg_num_tokens,
-                                 get_available_resources)
+                                 get_available_resources, _ray_init)
```

Comment:
as we are importing and using `_ray_init` in a few places, wouldn't it be better making it properly available without the `_` in the name?